src/c14n-canonicalization.ts

```diff
@@ -1,12 +1,14 @@
-const utils = require("./utils");
-
-/**
- * @type { import("../index.d.ts").CanonicalizationOrTransformationAlgorithm}
- */
-class C14nCanonicalization {
-  constructor() {
-    this.includeComments = false;
-  }
+import {
```

Comment:
nit - let's limit to import type, so that the codebase is slightly more durable to a variety of build approaches
```suggestion
import type {
```

see also elsewhere